packages/app/src/pages/landelijk/ziekenhuis-opnames.tsx

```diff
@@ -244,6 +253,28 @@ const IntakeHospital: FCWithLayout<typeof getStaticProps> = (props) => {
           metadata={{
             source: text.bronnen.lnaz,
           }}
+          formatTooltip={(values) => {
+            const value = values[0];
+            const isInaccurateValue = value.__date < lcpsOldDataRange[0];
```

Comment:
don't you mean 1 june 2020?
```suggestion
            const isInaccurateValue = value.__date < lcpsOldDataRange[1];
```

Reply:
Haha good catch!